editor/doc_tools.cpp

```diff
@@ -1696,8 +1744,6 @@ Error DocTools::save_classes(const String &p_default_path, const HashMap<String,
 		_write_method_doc(f, "annotation", c.annotations);
 
 		if (!c.theme_properties.is_empty()) {
-			c.theme_properties.sort();
```

Comment:
Removed the sorts here as they were mostly unnecessary and done after already sorting, and moved that sorting into the generation where it does not already apply